vignettes/pkgnet-gallery.Rmd

```diff
@@ -8,55 +8,147 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
-Welcome to the **pkgnet** gallery!   Click one any item below to see an example report.  Go to the [gallery github page](https://github.com/UptakeOpenSource/pkgnet-gallery) to contribute. 
-
-<!--html_preserve-->
-<table>
-	<tr valign="top">
-		<td width="33%">bingo<br>
-			<a href="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/bingo/bingo.html">
-				<img width="300" src="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/bingo/bingo.png">
-			</a>
-		</td>
-		<td width="33%">data.table<br>
-			<a href="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/data_table/data_table.html">
-				<img width="300" src="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/data_table/data_table.png">
-			</a>
-		</td>
-		<td width="33%">ggplot2<br>
-			<a href="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/ggplot2/ggplot2.html">
-				<img width="300" src="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/ggplot2/ggplot2.png">
-			</a>
-		</td>
-	</tr>
-	<tr>
-		<td width="33%">lubridate<br>
-			<a href="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/lubridate/lubridate.html">
-				<img width="300" src="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/lubridate/lubridate.png">
-			</a>
-		</td>
-		<td width="33%">pipecleaner<br>
-			<a href="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/pipecleaner/pipecleaner.html">
-				<img width="300" src="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/pipecleaner/pipecleaner.png">
-			</a>
-		</td>
-		<td width="33%">pkgnet<br>
-			<a href="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/pkgnet/pkgnet.html">
-				<img width="300" src="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/pkgnet/pkgnet.png">
-			</a>
-		</td>
-	</tr>
-	<tr>
-		<td width="33%">pkgnet (Vignette)<br>
-			<a href="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/pkgnet-vignette/pkgnet-vignette.html">
-				<img width="300" src="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/pkgnet/pkgnet.png">
-			</a>
-		</td>
-		<td width="33%">updraft<br>
-			<a href="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/updraft/updraft.html">
-				<img width="300" src="https://uptakeopensource.github.io/pkgnet-gallery/exhibits/updraft/updraft.png">
-			</a>
-		</td>
-	</tr>
-</table>
-<!--/html_preserve-->
+Welcome to the **pkgnet** gallery!   Click any item below to see an example report.  Go to the [gallery github page](https://github.com/UptakeOpenSource/pkgnet-gallery) to contribute. 
+
+```{r exhibit_data, include = FALSE}
```

Comment:
@jayqi should we add `library()` statements within this script to make package needs explicit.  I know it will really only be authors and maintainers generating the gallery.  However, for clarity, to round out this example, how about those statements?

Reply:
or in pkgnet_build_gallery.R

Reply:
I will add in subsequent PR.  This does not affect the submission of an exhibit, and I would like to commit the changes that do today.